tests/test_api.py

```diff
@@ -117,6 +117,27 @@ def test_lowercase_and_uppercase_in_argument() -> None:
 
     assert matched
 
+def test_setvar_declarations() -> None:
+    """ Example test various setvar view in the wild """
+    rule_text = """
+    SecRule REQUEST_URI "@rx .?" \
+    "id:1234,\
+    phase:1,\
+    pass,\
+    t:lowercase,\
+    setvar:'tx.false-positive-report-plugin_smtp_subject=<server_hostname> - <host_header>: False positive report from CRS',\
+    setvar:'tx.false-positive-report-plugin_filter_ignore_id=949110 959100 980130 980140'"
+    """
+
+    matched = False
+    uppercase_regex = re.compile(r"[A-Z]")
+    parsed_rule = parser.process_from_str(rule_text)
+    for rule in parsed_rule.rules:
+        assert rule.__class__.__name__ == "SecRule"
+        // TODO
```

Comment:
In Python, you can't use `//` for a comment. Please remove this line (otherwise it does not contain any information)